scout/commands/load/research.py

```diff
@@ -120,6 +120,16 @@ def research(case_id, institute, force):
                 rank_treshold=default_threshold,
             )
 
+        if case_obj["vcf_files"].get("vcf_mei_research"):
```

Comment:
Can this function be refactored so there are less IFs and repeated code? perhaps importing file types from constants?

<img width="696" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/2f6b9caf-2919-4f80-9275-e4651522831d">